brd-ios/breadwallet/src/Redesign/Views/SwapCurrencyView.swift

```diff
@@ -0,0 +1,259 @@
+// 
+//  SwapCurrencyView.swift
+//  breadwallet
+//
+//  Created by Kenan Mamedoff on 05/07/2022.
+//  Copyright © 2022 Fabriik Exchange, LLC. All rights reserved.
+//
+//  See the LICENSE file at the project root for license information.
+//
+
+import UIKit
+
+class SwapCurrencyView: UIView {
+    var didChangeFiatAmount: ((SwapMainModels.Amounts.CurrencyData?) -> Void)?
+    var didChangeCryptoAmount: ((SwapMainModels.Amounts.CurrencyData?) -> Void)?
+    
+    private lazy var currencyContainerView: UIView = {
+        let view = UIView()
+        return view
+    }()
+    
+    private lazy var currencyHeaderStackView: UIStackView = {
+        let view = UIStackView()
+        view.axis = .horizontal
+        view.distribution = .equalSpacing
+        return view
+    }()
+    
+    lazy var iHaveLabel: FELabel = {
```

Comment:
make all private and handle animations of this views subviews inside here instead of the parent view